inc/url_replacer.php

```diff
@@ -251,16 +251,6 @@ private function normalize_image( $url, $original_url, $args, $is_uploaded = fal
 			}
 		}
 
-		if ( $args['width'] > 0 && $args['height'] > 0 ) {
-			list( $args['width'], $args['height'] ) = wp_constrain_dimensions( $args['width'], $args['height'], $this->max_width, $this->max_height );
-		} elseif ( $args['width'] > 0 ) {
-			$args['width'] = $args['width'] > $this->max_width ? $this->max_width : $args['width'];
-		} elseif ( $args['height'] > 0 ) {
-			$args['height'] = $args['height'] > $this->max_height ? $this->max_height : $args['height'];
-		}
-		if ( isset( $args['resize'], $args['resize']['gravity'] ) && $this->settings->is_smart_cropping() ) {
```

Comment:
@abaicus this shouldnt be removed since is related to smart crop, no?

Reply:
@selul yes, indeed. Sorry for that! Thank you for letting me know!